workloads/mysql/Makefile

```diff
@@ -69,6 +95,12 @@ privileges:
 uninstall-mysql:
 	rm -rf /var/run/mysqld /tmp/mysql-data /var/lib/mysql-files *.sig *.manifest.sgx *.manifest
 
+.PHONY: uninstall-mysql-rhel
+uninstall-mysql-rhel:
+	sudo dnf -y remove mysql-server sysbench
```

Comment:
Why are you uninstalling the packages in rhel and not in Ubuntu? as part of other examples we don't do uninstallation, as anyways the docker container will be removed post the run. This will also allow you to merge uninstall-mysql with uninstall-mysql-rhel